src/rgw/rgw_rest_s3.cc

```diff
@@ -2103,6 +2103,7 @@ int RGWCopyObj_ObjStore_S3::get_params()
       attrs_mod = RGWRados::ATTRSMOD_NONE; // default for intra-zone_group copy
     } else {
       ldout(s->cct, 0) << "invalid metadata directive" << dendl;
+      s->err.message = "Unknown metadata directive.";
```

Comment:
Look at the other functions that does the same. The `s->err.message` is assigned first, which is fed to `ldout`. I think you could make this change similar.